### PR TITLE
Prompts doc rewrite

### PR DESCRIPTION
**Pull request by LinaLam**
- combined the prompts **introduction** and **get started** page to one file 
- replaced the **Prompts** folder structure with a single file on the left nav
- added a card for **Prompts** from the **Welcome** > **Introduction** page

### CODE REVIEW

**Comment by LinaLam** [comment]
screencapture-localhost-3000-features-prompts-2024-05-07-20_58_19

File: docs/mint.json [hunk 1]
```diff
@@ -115,10 +115,7 @@
     {
       "group": "Features",
       "pages": [
-        {
-          "group": "Prompts",
-          "pages": ["features/prompts/intro", "features/prompts/quick-start"]
```
Comment (chitalian) [inline docs/mint.json:120]: can we remove these files as well?

File: docs/getting-started/integration-method/gateway-fallbacks.mdx [hunk 2]
```diff
@@ -1,14 +1,21 @@
 ---
 title: "Gateway Fallback (Beta)"
-description: "With the instability of some APIs and the similarity of many providers, we have developed a unified gateway. This gateway allows you to utilize any provider through a single endpoint."
```
Comment (chitalian) [inline docs/getting-started/integration-method/gateway-fallbacks.mdx:3]: were these changes supposed to be in this PR?

File: docs/getting-started/integration-method/gateway-fallbacks.mdx
Comment (LinaLam) [inline docs/getting-started/integration-method/gateway-fallbacks.mdx:1]: just reverted changes to match main - this file was not supposed to be part of this PR

**Pull request merged**